psycopg/psycopg/pq/pq_ctypes.py

```diff
@@ -240,10 +240,14 @@ def backend_pid(self) -> int:
 
     @property
     def needs_password(self) -> bool:
+        """True if the connection authentication method required a password,
+        but none was available.
+        """
         return bool(impl.PQconnectionNeedsPassword(self._pgconn_ptr))
 
     @property
     def used_password(self) -> bool:
+        """True if the connection authentication method used a password."""
```

Comment:
Maybe we can add the links to the libpq documentation? There is a `:pq:` interpreted role for it, see `get_cancel()` docstring for details.

Reply:
Indeed, thanks for the tip!